lute/std/libs/test/types.luau

```diff
@@ -3,6 +3,7 @@
 -- Standard test types library for Luau
 
 -- Test Reporting
+local failure = require("./failure")
 local assert = require("./assert")
```

Comment:
unless there's a good reason not to, i think we should try to define all types in `types.luau`, and have `failure` and `assert` import those types from `types.luau`